js/apps/system/_admin/aardvark/APP/frontend/js/routers/versionCheck.js

```diff
@@ -44,93 +44,13 @@
-  var getInformation = function () {
-    var nVer = navigator.appVersion;
-    var nAgt = navigator.userAgent;
-    var browserName = navigator.appName;
-    var fullVersion = '' + parseFloat(navigator.appVersion);
-    var majorVersion = parseInt(navigator.appVersion, 10);
-    var nameOffset,verOffset,ix;
-
-    if ((verOffset = nAgt.indexOf('Opera')) !== -1) {
-      browserName = 'Opera';
-      fullVersion = nAgt.substring(verOffset + 6);
-      if ((verOffset = nAgt.indexOf('Version')) !== -1) {
-        fullVersion = nAgt.substring(verOffset + 8);
-      }
-    } else if ((verOffset = nAgt.indexOf('MSIE')) !== -1) {
-      browserName = 'Microsoft Internet Explorer';
-      fullVersion = nAgt.substring(verOffset + 5);
-    } else if ((verOffset = nAgt.indexOf('Chrome')) !== -1) {
-      browserName = 'Chrome';
-      fullVersion = nAgt.substring(verOffset + 7);
-    } else if ((verOffset = nAgt.indexOf('Safari')) !== -1) {
-      browserName = 'Safari';
-      fullVersion = nAgt.substring(verOffset + 7);
-      if ((verOffset = nAgt.indexOf('Version')) !== -1) {
-        fullVersion = nAgt.substring(verOffset + 8);
-      }
-    } else if ((verOffset = nAgt.indexOf('Firefox')) !== -1) {
-      browserName = 'Firefox';
-      fullVersion = nAgt.substring(verOffset + 8);
-    } else if ((nameOffset = nAgt.lastIndexOf(' ') + 1) < (verOffset = nAgt.lastIndexOf('/'))) {
-      browserName = nAgt.substring(nameOffset, verOffset);
-      fullVersion = nAgt.substring(verOffset + 1);
-      if (browserName.toLowerCase() === browserName.toUpperCase()) {
-        browserName = navigator.appName;
-      }
-    }
-    if ((ix = fullVersion.indexOf(';')) !== -1) {
-      fullVersion = fullVersion.substring(0, ix);
-    }
-    if ((ix = fullVersion.indexOf(' ')) !== -1) {
-      fullVersion = fullVersion.substring(0, ix);
-    }
-
-    majorVersion = parseInt('' + fullVersion, 10);
-
-    if (isNaN(majorVersion)) {
-      fullVersion = '' + parseFloat(navigator.appVersion);
-      majorVersion = parseInt(navigator.appVersion, 10);
-    }
-
-    return {
-      browserName: browserName,
-      fullVersion: fullVersion,
-      majorVersion: majorVersion,
-      appName: navigator.appName,
-      userAgent: navigator.userAgent
-    };
-  };
-  */
-
-  /*
-  var getOS = function () {
-    var OSName = 'Unknown OS';
-    if (navigator.appVersion.indexOf('Win') !== -1) {
-      OSName = 'Windows';
-    }
-    if (navigator.appVersion.indexOf('Mac') !== -1) {
-      OSName = 'MacOS';
-    }
-    if (navigator.appVersion.indexOf('X11') !== -1) {
-      OSName = 'UNIX';
-    }
-    if (navigator.appVersion.indexOf('Linux') !== -1) {
-      OSName = 'Linux';
-    }
-
-    return OSName;
-  };
-  */
-
   window.checkVersion = function () {
     // this checks for version updates
     $.ajax({
       type: 'GET',
       cache: false,
-      url: arangoHelper.databaseUrl('/_api/version'),
+      url: arangoHelper.databaseUrl('/_admin/status?overview=true&browser='
```

Comment:
![image](https://user-images.githubusercontent.com/1244163/72896910-0ed0c700-3d21-11ea-9e12-57f96a47c09f.png)

Maybe we should check here if `window.navigator.appName` and `window.navigator.appVersion` are both available.

Reply:
See: https://developer.mozilla.org/de/docs/Web/API/Window/navigator

Reply:
remover browser